addons/project/static/src/components/subtask_kanban_list/subtask_kanban_list.js

```diff
@@ -78,6 +78,7 @@ export class SubtaskKanbanList extends Component {
             parent_id: this.props.record.resId,
             project_id: this.props.record.data.project_id[0],
             user_ids: this.props.record.data.user_ids.resIds,
+            sequence: this.list.records[this.list.count - 1].data.sequence + 1,
```

Comment:
maybe searching the max sequence. 🤷🏼 

I don't know if we are sure the last one has the highest sequence.